### ISSUE TITLE
jaws new stage failes on malformed project's jaws.json

### ISSUE THREAD

**Issue by jodagm** [jaws-framework/JAWS#138]
the project's jaws.json is created without a 'project' attrbute, which is required by new stage cli

new_stage_region command validates that there is a project attribute:

if (!_this._JAWS._meta.projectJson.project || !_this._JAWS._meta.projectJson.stages) 
and throw JawsError

**Comment by doapp-ryanp** [comment]
This is a bug. the `project` attribute was removed with the v1.0 refactor.  I will go through the new region and new stage code here today and make sure its upgraded to v1.0 changes.

**Comment by ac360** [comment]
Fixed a while ago
